configure.ac

```diff
@@ -630,6 +630,12 @@ case "$host_cpu" in
     c_targetcpu="m68k"
     c_unalignedmemory=yes
     ;;
+   loongarch64*)
+    AC_DEFINE(C_TARGETCPU,LOONGARCH64)
+    AC_MSG_RESULT(Loongson 3A5000+)
```

Comment:
Maybe just "LoongArch"? Even if just considering Loongson products, this expression excludes the 2K product line, which may not be what you mean.